uniffi/src/lib.rs

```diff
@@ -179,7 +179,7 @@ pub unsafe trait ViaFfi: Sized {
     /// This trait method can be used for sending data from rust to the foreign language code,
     /// in cases where we're not able to use a special-purpose FFI type and must fall back to
     /// sending serialized bytes.
-    fn write(&self, buf: &mut Vec<u8>);
+    fn write(self, buf: &mut Vec<u8>);
```

Comment:
The docs for the `lower` method have an explicit note to explain why it takes an owned self, could you please add similar docs to the `write`method here for completeness?